src/js/components/covid19/awardSpendingAgency/AwardSpendingAgency.jsx

```diff
@@ -33,11 +35,18 @@ const overviewData = [
 ];
 
 const AwardSpendingAgency = () => {
-    const [awardCount, setAwardCount] = useState(null);
+    const [agencyCount, setAgencyCount] = useState(null);
     const [awardOutlays, setAwardOutlays] = useState(null);
     const [awardObligations, setAwardObligations] = useState(null);
     const [numberOfAwards, setNumberOfAwards] = useState(null);
-    const [activeTabLabel, setActiveTabLabel] = useState(awardSpendingAgencyTableTabs[0].label);
+
+    const [activeTab, setActiveTab] = useState(
+        {
+            internal: awardSpendingAgencyTableTabs[0].internal,
+            subtitle: awardSpendingAgencyTableTabs[0].subtitle
+        }
+    );
+
     const defCodes = useSelector((state) => state.covid19.defCodes);
 
     // TODO - Remove hard coded values
```

Comment:
Let's go ahead and remove the hard-coded values and request these counts from the [v2/disaster/agency/count](https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/disaster/agency/count.md) mock API. I talked to Tony and ultimately we hope to add single endpoint with all these counts, but for now we'll make multiple calls. Example in the CFDA tabs branch [here](https://github.com/fedspendingtransparency/usaspending-website/blob/mod/dev-5458-covid-cfda-award-tabs/src/js/components/covid19/assistanceListing/SpendingByCFDA.jsx#L46)